effective-image-guide/readme.md

```diff
@@ -0,0 +1,56 @@
+>**The goal of this guide:** to explore what makes an image effective in relation to an article.
+
+# Sciencing the $@%* out of images
+
+When used correctly, effective visuals are extremely powerful to communicate an appropriate message. Let's look at some of the science behind visual content and what makes an image effective.
+
+## First off: Why visual content is so important
+
+Here are some telling stats as researched by [Buffer's](https://buffer.com/)  [Kevan Lee](https://blog.bufferapp.com/author/kevanlee) on just how big visual content has become and how it’s changing the way we all approach marketing:
+
+**63% of social media is made up of images.** That means nearly two-thirds of the updates you see on social media are visual content, according to a [Citrix report](http://www.slideshare.net/citrix/citrix-mobileusage) from 2014.
+
+
+**Nearly half of all Internet users have re-posted a photo or video they found online.** An equally intriguing stat from the same [2013 Pew Research Study](http://www.pewinternet.org/files/old-media//Files/Reports/2013/PIP_Photos%20and%20videos%20online_102813.pdf) is that 54% of all Internet users have posted an original photo or video that they personally have created.
```

Comment:
All internet users are: All American users, because it's an American study. Check page 3 *about this survey*